docs/release-notes/release-notes.md

```diff
@@ -2,6 +2,14 @@
 
 This guide provides notes for major version releases. These notes may be helpful for users when upgrading from previous major versions.
 
+## v4.0.0
+
+### Node Operators (v4.0.0)
+
+#### Multiplexer
```

Comment:
nit: perhaps add a link to redirect it to the multiplexer docs for those that want more info